netutils/rtptools/Kconfig

```diff
@@ -0,0 +1,139 @@
+#
+# For a description of the syntax of this configuration file,
+# see the file kconfig-language.txt in the NuttX tools repository.
+#
+
+config NETUTILS_RTPTOOLS
+	bool "Enable RTP Tools"
+	default n
+	---help---
+		RTP Tools is a set of small applications
+		that can be used for processing RTP data.
+
+if NETUTILS_RTPTOOLS
+
+config NETUTILS_RTPTOOLS_SPECIFIC_VERSION
+	bool "Define a version tag to checkout"
+	default n
+	---help---
+		Set a specific version tag to checkout. Otherwise,
+		master branch will be used.
```

Comment:
Should it be better always using a last tested version? Using master branch is always dangerous and prone to crash. It could be like the LVGL, use the latest version and let the user to select a different version

Reply:
Hi @acassis ! I updated the MR. Now it's ready to be merged and I fixed this issue, indicating a version tag/hash as usual.

Could you please take a look at it?